ruby31-plus-devkit/plan.ps1

```diff
@@ -0,0 +1,51 @@
+$pkg_name="ruby31-plus-devkit"
+$pkg_origin="chef"
+$pkg_version="3.1.2"
+$pkg_revision="1"
+$pkg_maintainer="maintainers@chef.io"
+$pkg_license=@("Apache-2.0")
+$pkg_source="https://github.com/oneclick/rubyinstaller2/releases/download/RubyInstaller-${pkg_version}-${pkg_revision}/rubyinstaller-devkit-${pkg_version}-${pkg_revision}-x64.exe"
+$pkg_shasum = "5f0fd4a206b164a627c46e619d2babbcafb0ed4bc3e409267b9a73b6c58bdec1"
+$pkg_bin_dirs=@(
+    "bin"
+    "/msys64/ucrt64/bin"
+    "/msys64/usr/bin"
+)
+$ruby_abi_version = [RegEx]::Replace($pkg_version, "\.\d+$", ".0")
+
+function Invoke-SetupEnvironment {
+    Push-RuntimeEnv -IsPath "GEM_PATH" "$pkg_prefix/lib/ruby/gems/$ruby_abi_version"
+}
+
+function Invoke-Unpack {
+    Write-BuildLine "Unpacking $pkg_filename"
+    Start-Process "$HAB_CACHE_SRC_PATH/$pkg_filename" -Wait -ArgumentList "/SP- /NORESTART /VERYSILENT /SUPPRESSMSGBOXES /NOPATH /DIR=$HAB_CACHE_SRC_PATH/$pkg_dirname"
+}
+
+function Invoke-Build {
+    Write-BuildLine "** Launch msys2 once in order to initialize the install"
+    Invoke-Expression -Command "$HAB_CACHE_SRC_PATH/$pkg_dirname/msys64/msys2_shell.cmd -defterm -no-start -c exit" -Verbose
+    # The path separator in the -ILike value below must be a backslash for it will match properly
+    Get-Process | Where-Object Path -ILike "$HAB_CACHE_SRC_PATH\$pkg_dirname\*" | Stop-Process -Force
+}
+
+function Invoke-Install {
+    Write-BuildLine "** Copying files to the package location"
+    Copy-Item "$HAB_CACHE_SRC_PATH/$pkg_dirname/*" "$pkg_prefix" -Recurse -Force
+
+    Write-BuildLine "** Include git because why not at this point."
+    # Invoke-Expression -Command "$pkg_prefix/bin/ridk.cmd exec pacman -S --noconfirm git" -Verbose
```

Comment:
if this is not needed then it should be deleted.

Reply:
Done